docs/README.md

```diff
@@ -2,11 +2,53 @@
 
 The Docker plugin is intended to be used for managing docker images through Packer:
 
-### Builders:
+## Installation
```

Comment:
This will be overwritten by terraform. We should update the template in the terraform configuration.

Reply:
No, wait. Forget about that, I just saw the terraform configuration is already updated with this.